### PR TITLE
update README.md in autoware_joy_controller

### PR DESCRIPTION
**Pull request by takayuki5168**
## Description

add README.md of autoware_joy_controller

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [ ] Read [pull request guidelines][pull-request-guidelines]
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] (Optional) Unit tests have been written for new behavior
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write [release notes][release-notes]

## CI Checks

- **Build and test for PR / build-and-test-pr**: Required to pass before the merge.
- **Build and test for PR / clang-tidy-pr**: NOT required to pass before the merge. It is up to the reviewer(s). Found false positives? See the [guidelines][clang-tidy-guidelines].
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[clang-tidy-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/ClangTidyGuideline/
[coding-guidelines]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/1194394777/T4
[pull-request-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/PullRequestGuideline/
[release-notes]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/563774416
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

**Comment by takayuki5168** [comment]
@tkimura4 @taikitanaka3 
Could you review?

https://github.com/tier4/AutowareArchitectureProposal.iv/blob/b69e7b29bc1f5163940666c2d4a619bbb1e3879e/control/autoware_joy_controller/README.md

File: control/autoware_joy_controller/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# autoware_joy_controller
+
+## Role
+
+`autoware_joy_controller` operate a vehicle by a joy controller via Autoware (e.g. steering wheel, shift, turn signal, engage) by several kinds of joy sticks.
```
Comment (tkimura4) [inline control/autoware_joy_controller/README.md:5]: ~  is the packages to convert joy msg to autoware command (e.g. ~~ )for a vehicle ?

File: control/autoware_joy_controller/README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+| Name               | Type                    | Description                       |
+| ------------------ | ----------------------- | --------------------------------- |
+| `~/input/joy`      | sensor_msgs::msg::Joy   | joy controller command            |
+| `~/input/odometry` | nav_msgs::msg::Odometry | ego vehicle odometry to get twist |
+
+### Output topics
+
+| Name                                | Type                                                     | Description                              |
+| ----------------------------------- | -------------------------------------------------------- | ---------------------------------------- |
+| `~/output/control_command`          | autoware_auto_control_msgs::msg::AckermannCommandStamped | lateral and longitudinal control command |
```
Comment (taikitanaka3) [inline control/autoware_joy_controller/README.md:20]:
```suggestion
| `~/output/control_command`          | autoware_auto_control_msgs::msg::AckermannControlCommand| lateral and longitudinal control command |
```

File: control/autoware_joy_controller/README.md (hunk 1 above)
Comment (takayuki5168) [reply control/autoware_joy_controller/README.md]: https://github.com/tier4/AutowareArchitectureProposal.iv/pull/593/commits/1842070d0c115ba618c145e23c14b7747e19c9a0
Did it. Thanks

**Review by tkimura4 — APPROVED** [review]
LGTM

**Pull request merged**